com.unity.render-pipelines.universal/Runtime/NativeRenderPass.cs

```diff
@@ -174,13 +174,13 @@ internal void SetNativeRenderPassMRTAttachmentList(ScriptableRenderPass renderPa
                         // if this is the current camera's last pass, also check if one of the RTs is the backbuffer (BuiltinRenderTextureType.CameraTarget)
                         isLastPassToBB |=  pass.isLastPass && (pass.colorAttachments[i] == BuiltinRenderTextureType.CameraTarget);
 
-                        int existingAttachmentIndex = FindAttachmentDescriptorIndexInList(pass.colorAttachments[i], m_ActiveColorAttachmentDescriptors);
+                        int existingAttachmentIndex = FindAttachmentDescriptorIndexInList(pass.overrideCameraTarget ? pass.colorAttachments[i] : m_CameraColorTarget, m_ActiveColorAttachmentDescriptors);
 
                         if (existingAttachmentIndex == -1)
                         {
                             // add a new attachment
                             m_ActiveColorAttachmentDescriptors[currentAttachmentIdx] = currentAttachmentDescriptor;
-                            m_ActiveColorAttachmentDescriptors[currentAttachmentIdx].ConfigureTarget(pass.colorAttachments[i],  (clearFlag & ClearFlag.Color) == 0, true);
+                            m_ActiveColorAttachmentDescriptors[currentAttachmentIdx].ConfigureTarget(pass.overrideCameraTarget ? pass.colorAttachments[i] : m_CameraColorTarget,  (pass.clearFlag & ClearFlag.Color) == 0, true);
 
```

Comment:
since this line is repeated probably would be better to store the result camera in a. shared variable to avoid having to update the logic in 2 different places in the future?

pass.overrideCameraTarget ? pass.colorAttachments[i] : m_CameraColorTarget

Reply:
fixed it